logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/e2ei/usecase/DownloadE2eiCertificateUseCase.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.e2ei.usecase
+
+/**
+ * This use case is used to download e2ei certificate
+ */
+class DownloadE2eiCertificateUseCase {
```

Comment:
suggestion: download is not correct, since we're not downloading the certificate at all, we only getting it from CoreCrypto [it's already there!] and for this usecase, we should pass the clientId in order to get the certificate for that specific client.